apps/web/examples/newPage/example.service.js

```diff
@@ -0,0 +1,52 @@
+// Import request: this formats requests to the api
+import { get, patch, post } from '../../src/server/lib/request.js';
+
+/** @typedef {import('./example.types').Example} Example */
+/**
+ * @returns {Promise<Array<Example>>} TODO: change Example to dataType returned by Promise
```

Comment:
Just checking, is the TODO intended to be left in the example?

Reply:
Yes, I will be adding ToDos so that when it is copied and pasted, you know what needs to be modified.
Or does that sound like a bad idea?